scripts/nockma-stdlib-parser.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# Usage:
+# chmod +x nockma-stdlib-parser.sh
+# ./nockma-stdlib-parser < anoma.hoon
+
+# Use grep to find lines matching the pattern and awk to format the output
+# Reads from stdin
+grep -oP '\+\+  \K\w+\s+::\s+\+\d+' | awk '{gsub("::  \\+", ""); print}'
```

Comment:
What does this script do? There should be a comment.

Reply:
Added a comment to the script